src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -40,10 +40,12 @@
     &-story {
       padding-right: 70px;
       padding-left: 42px;
-      textarea[class='f8-quickadd-desc'] {
-        max-height: 100px;
-        height: 26px !important;
-        padding-top: 0;
+      textarea.f8-quickadd-desc {
+        bottom: -25px;
+        max-height: 200px;
+        max-width: 407px;
```

Comment:
max-width is not needed

Reply:
![screenshot from 2017 10 17 00 24 31](https://user-images.githubusercontent.com/12949454/31629626-d71d42ba-b2d1-11e7-98e5-c9480ca5aaae.png)

^^This is why max-width is necessary. The description field is a `col-md-6` and title field is also a `col-md-6` Without max-width, the description field overlaps with the buttons.